### PR TITLE
Recipe for abawaca binning tool

### PR DESCRIPTION
**Pull request by keuv-grvl**
- [x] I have read the guidelines above.
- [x] This PR adds a new recipe.
- [ ] This PR updates an existing recipe.
- [ ] This PR does something else (explain below).

The [abawaca repo](https://github.com/CK7/abawaca) does not state any licence, but author confirms it is open source (https://github.com/CK7/abawaca/issues/3).

### CODE REVIEW

File: recipes/abawaca/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/CK7/abawaca/archive/v{{ version }}.tar.gz
+  sha256: 57465bb291c3a9af93605ffb11d704324079036205e5ac279601c9e98c467529
+
+build:
+  number: 0
+
+requirements:
+  build:
+
```
Comment (bgruening) [inline recipes/abawaca/meta.yaml]: You are compiling from source here, correct? Please include gcc, llvm as stated in our Guidelines.

File: recipes/abawaca/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  run:
+
+about:
+  home: 'https://github.com/CK7/abawaca'
+  license: "open source"
+  summary: "abawaca is a binning program for metagenomics"
+
+extra:
+  maintainers:
+    - keuv-grvl
```
Comment (bgruening) [inline recipes/abawaca/meta.yaml]: Like this!

**Comment by bgruening** [comment]
Thanks for you contribution, please add GCC and LLVM as your build requirements according to our Guidelines.

File: recipes/abawaca/meta.yaml (hunk 1 above)
Comment (keuv-grvl) [inline recipes/abawaca/meta.yaml]: Done

**Comment by bgruening** [comment]
@keuv-grvl please poke me with a stick if I forget about a PR.
Or feel free to merge it by your own.

**Comment by keuv-grvl** [comment]
I also forgot it...
Thank you for merging this PR!

**Pull request merged**